source/faq.txt

```diff
@@ -734,21 +374,20 @@ Python 3, you must pass ``encoding='latin-1'`` to ``pickle.loads``:
    >>> pickle.dumps(oid)
    'ccopy_reg\n_reconstructor\np0\n(cbson.objectid\...'
 
-   Python 3.7.9 (v3.7.9:13c94747c7, Aug 15 2020, 01:31:08)
-   [Clang 6.0 (clang-600.0.57)] on darwin
-   Type "help", "copyright", "credits" or "license" for more information.
+   # Python 3.7
    >>> import pickle
    >>> pickle.loads(b'ccopy_reg\n_reconstructor\np0\n(cbson.objectid\...', encoding='latin-1')
    ObjectId('4f919ba2fba5225b84000000')
 
-If you need to pickle ObjectIds using Python 3 and unpickle them using Python 2,
-you must use ``protocol <= 2``:
+If you pickled an ``ObjectID`` in Python 2, and want to unpickle it in Python 3,
+you must pass the ``protocol <= 2`` option to the ``pickle.dumps()`` method.
```

Comment:
This line says to pass `protocol <= 2` to the method, but in the highlighted code block it passes `protocol=2`. I'm not sure which one is correct, or if I'm misunderstanding something with it, but we should probably clean that up